shared/teams/team/subteam-banner.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+import * as React from 'react'
+import {Box, Icon, Text} from '../../common-adapters'
+import {globalColors, globalMargins, globalStyles, isMobile} from '../../styles'
+
+export type Props = {
+  onReadMore: () => void,
+  onHideBanner?: () => void,
+}
+
+const Banner = ({onReadMore, onHideBanner}: Props) => (
+  <Box
+    style={{
+      ...(isMobile
```

Comment:
maybe better to make this a const so its not so ugly in the jsx w/ prettier

Reply:
done